Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM ubuntu:focal
+USER root
+WORKDIR /data
+ARG ROOTDATADIR=/data
+ARG RPCUSERNAME=user
+ARG RPCPASSWORD=pass
+ARG DGBVERSION=7.17.2
+ARG ARCH=x86_64
+
+ARG MAINP2P=12024
+ARG MAINRPC=14022
+ARG TESTP2P=12026
+ARG TESTRPC=14023
+
+# You can confirm your timezone by setting the TZ database name field from:
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+ARG LOCALTIMEZONE=Pacific/Auckland
```

Comment:
@ChillingSilence and @SmartArray , will this time zone have any negative impact on getting time/dates from the node?  Should this be set to UTC?

Reply:
It shouldn't really matter but I agree and it can be set to UTC.

Reply:
@ChillingSilence can you update this to https://en.wikipedia.org/wiki/Coordinated_Universal_Time ?  I think the code should be:

`ARG LOCALTIMEZONE=UTC`

@SmartArray , thoughts?

Reply:
Updated